README.md

```diff
@@ -227,16 +227,16 @@ S-GPT is already in Shortcuts gallery. Easily ask any questions to chatGPT via S
 
 **Translation:**
 
-| Language | Contributor              | Language  | Contributor    |
-|----------|--------------------------|:----------|:---------------|
-| German   | @C0d3Br3aker             | Italian   | @bellaposa     |
-| Croatian | @milotype                | Russian   | @kirillyakopov |
-| Turkish  | @berkbatuhans            | Spanish   | @kant          |
-| Polish   | @kpacholak               | Japanese  | @ShogoKoyama   |
-| Dutch    | Alex                     | Somali    | @abdorizak     |
-| Filipino | Rosel                    | Korean    | @iosdevted     |
-| French   | @BtKent and Ange Lefrère | Ukrainian | @andryua       |
-| Slovak   | @Svec-Tomas              |           |                |
+| Language | Contributor              | Language        | Contributor    |
+|----------|--------------------------|:----------------|:---------------|
+| German   | @C0d3Br3aker             | Italian         | @bellaposa     |
+| Croatian | @milotype                | Russian         | @kirillyakopov |
+| Turkish  | @berkbatuhans            | Spanish         | @kant          |
+| Polish   | @kpacholak               | Japanese        | @ShogoKoyama   |
+| Dutch    | Alex                     | Somali          | @abdorizak     |
+| Filipino | Rosel                    | Korean          | @iosdevted     |
+| French   | @BtKent and Ange Lefrère | Ukrainian       | @andryua       |
+| Slovak   | @Svec-Tomas              | Portuguese (BR) | @EvertonCa     |
```

Comment:
Please also add Portugusee (BR) in `Supported Languages 🇺🇳` Section, scroll up on README, you will find it.